Src/PerformanceCollector/Shared/Implementation/QuickPulse/IQuickPulseServiceClient.cs

```diff
@@ -2,7 +2,8 @@
 {
     using System;
     using System.Collections.Generic;
-    using System.Security.Cryptography.X509Certificates;
+
+    using Microsoft.ApplicationInsights.Extensibility.Filtering;
```

Comment:
Same reason as the shared project is called "filtering" - the original tag by which this functionality was introduced was "filtering telemetry to calculate metrics". Can you think of a better name? This one does seem a bit outdated at this point.